libsolidity/ast/ASTUtils.h

```diff
@@ -38,4 +38,8 @@ bool isConstantVariableRecursive(VariableDeclaration const& _varDecl);
 /// Returns the innermost AST node that covers the given location or nullptr if not found.
 ASTNode const* locateInnermostASTNode(int _offsetInFile, SourceUnit const& _sourceUnit);
 
+/// @returns @a _expr itself, in case it is not an unary tuple expression. Otherwise it descends recursively
+/// into unary tuples and returns the contained expression.
+Expression const* resolveUnaryTuples(Expression const* _expr);
```

Comment:
This does nothing for any unary tuples nested somewhere deeper in the expression. Just the outer ones, right?
```suggestion
Expression const* resolveOuterUnaryTuples(Expression const* _expr);
```